frontend/packages/console-app/src/components/guided-tours/utils/guided-tour-typings.ts

```diff
@@ -0,0 +1,10 @@
+export type GuidedTourCatalogItem = {
+  iconURL: string;
+  altIcon?: string;
+  name: string;
+  status: string;
+  active?: boolean;
+  duration: number;
+  description: string;
+  prerequisites?: string[];
+};
```

Comment:
There needs to be a guided tour specification type that defines the fields that a guided tour author must provide. This spec should not contain `status` and `active` properties.

Then there should be another type for capturing the state of a guided tour. This is the state that should be serialized to track progress and resume the tour. This state shouldn't contain all the tour information.

Combining the spec and the state is what's needed in the catalog item. The catalog item props should be defined in the component itself then as a combination of the two types.